packages/portal/src/components/search/SearchQueryBuilderRule.vue

```diff
@@ -6,104 +6,94 @@
+                  :title="$t(`search.advanced.tooltip.${component}`)"
+                  boundary-padding="0"
+                  placement="bottom"
+                />
+              </template>
+            </span>
+          </template>
+          <div
+            :id="`${id}-${component}`"
           >
-            {{ $t('search.advanced.input.field') }}
-            <template v-if="tooltips">
-              <b-button
-                :id="`select-field-tooltip-btn-${id}`"
-                class="icon-info-outline py-0 px-1 tooltip-button"
-                variant="light-flat"
-              />
-              <b-tooltip
-                :target="`select-field-tooltip-btn-${id}`"
-                :title="$t('search.advanced.tooltip.field')"
-                boundary-padding="0"
-                placement="bottom"
-              />
-            </template>
-          </span>
-        </template>
-        <b-form-select
-          :id="`select-field-${id}`"
-          v-model="field"
-          :options="selectFieldOptions"
-          :required="areAllRequired"
-          @change="(value) => handleFieldChange(value)"
-        />
-      </b-form-group>
-      <b-form-group
-        class="query-rule-form-group mr-lg-2"
-        :label-for="`select-modifier-${id}`"
-      >
-        <template #label>
-          <span
-            :id="`select-modifier-label-${id}`"
-            class="d-inline-flex align-items-center"
-          >
-            {{ $t('search.advanced.input.modifier') }}
-            <template v-if="tooltips">
-              <b-button
-                :id="`select-modifier-tooltip-btn-${id}`"
-                class="icon-info-outline py-0 px-1 tooltip-button"
-                variant="light-flat"
-              />
-              <b-tooltip
-                :target="`select-modifier-tooltip-btn-${id}`"
-                :title="$t('search.advanced.tooltip.modifier')"
-                boundary-padding="0"
-                placement="bottom"
-              />
-            </template>
-          </span>
-        </template>
-        <b-form-select
-          :id="`select-modifier-${id}`"
-          v-model="modifier"
-          :options="selectModifierOptions"
-          :required="areAllRequired"
-          @change="(value) => handleModifierChange(value)"
-        />
-      </b-form-group>
-      <b-form-group
-        class="query-rule-form-group"
-        :label-for="`search-term-${id}`"
-      >
-        <template #label>
-          <span
-            :id="`search-term-label-${id}`"
-            class="d-inline-flex align-items-center"
+            <b-form-input
+              v-if="component === 'term'"
+              v-model="term"
+              :data-qa="`advanced search query builder: ${component} control`"
+              :placeholder="$t('search.advanced.placeholder.term')"
+              :state="validations.term.state"
+              @change="(value) => handleRuleChange('term', value)"
+            />
+            <b-dropdown
```

Comment:
should dropdowns have their own component?
There's a bit of logic to these now.

Reply:
Done.